### PR TITLE
feat: Updated profile page

### PR DESCRIPTION
**Pull request by vvyn**
Changed profile page ui

### CODE REVIEW

File: pages/profile.tsx [hunk 1]
```diff
@@ -84,53 +87,103 @@ export default function ProfilePage() {
             />
```
Comment (DedsecKnight) [inline pages/profile.tsx:87]: Can we specify hacker's group here as well?

**Review by DedsecKnight — CHANGES_REQUESTED** [review]

File: pages/profile.tsx [hunk 2]
```diff
@@ -70,6 +70,9 @@ export default function ProfilePage({ userStatus }) {
     return <div></div>;
   }
 
+  console.log(user);
+  console.log(profile);
```
Comment (DedsecKnight) [inline pages/profile.tsx]: Remove this.

**Review by DedsecKnight — CHANGES_REQUESTED** [review]

**Review by DedsecKnight — APPROVED** [review]
lgtm :+1:

**Review by raywa04 — APPROVED** [review]
lgtm!

**Pull request merged**